lib/model/profile_M/profile_model.dart

```diff
@@ -0,0 +1,25 @@
+class Profile {
+  final String banner;
+  final String bio;
+  final String profilePicture;
+  final Map<String, dynamic> userName; // { 'profilename': 'JohnDoe', 'uniqueNumber': 123 }
+  final String profileName;
+  final double uniqueNumber;// since it has a larger range
```

Comment:
So when I changed this to int instead of double, my profile loads on my phone. Could you perhaps try changing it to int and see if it also still works on web?

Reply:
i changed it to an int and it still works on the web